code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1524,6 +1524,22 @@
 	REMOVE_TRAIT(L, TRAIT_RESISTHEAT, type)
 	return ..()
 
+/datum/reagent/zauker
+	name = "Zauker"
+	description = "An unstable gas that is toxic to all living beings"
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "90560B"
+	taste_description = "bitter"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED|REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/zauker/on_mob_metabolize(mob/living/L)
+	. = ..()
+	L.adjustBruteLoss(25)
+	L.adjustOxyLoss(5)
+	L.adjustFireLoss(8)
+	L.adjustToxLoss(8)
```

Comment:
using on_mob_metabolize() here will cause zauker to only deal damage when it first enters your system and not deal damage if you already have zauker in your system. it will also cause buckling and unbuckling someone from a stasis bed to re-proc this damage on the patient's next chem tick.

you probably want to use on_mob_life() here instead.

Reply:
I see will change it